src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -1251,12 +1251,15 @@ protected function getIndexSqlDefinition(Index $index, $tableName)
             return $ret;
         }, $columnNames);
 
+        $includedColumns = $index->getInclude() ? sprintf(' INCLUDE ("%s")', implode('","', $index->getInclude())) : '';
```

Comment:
Given the space in the query below, probably don't need the space before `INCLUDE` here